common/src/main/java/com/microsoft/hydralab/performance/PerformanceTestListener.java

```diff
@@ -0,0 +1,13 @@
+package com.microsoft.hydralab.performance;
+
+public interface PerformanceTestListener {
+    void testStarted(String testName);
+
+    void testSuccess(String testName);
+
+    void testFailure(String testName);
+
+    void testRunStarted();
```

Comment:
Could you provide some Javadoc here to specify what's the meaning of these methods?

For example, what's the different between testStarted and testRunStarted? How do you define testRun and test?

Reply:
Added